modules/command/common/export-command.js

```diff
@@ -0,0 +1,103 @@
+const Command = require('../command');
+const Models = require('../../../models');
+const { fork } = require('child_process');
+
+/**
+ * Increases approval for Bidding contract on blockchain
+ */
+class ExportCommand extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.logger = ctx.logger;
+        this.notifyError = ctx.notifyError;
+        this.importService = ctx.importService;
+        this.config = ctx.config;
+    }
+
+    async execute(command) {
+        const {
+            datasetId,
+            handlerId,
+            standardId,
+        } = command.data;
+
+        const importResult = await this.importService.getImport(datasetId);
+
+        if (importResult.error) {
+            await this.handleError(importResult.error);
+        } else {
+            const forked = fork('modules/worker/export-worker.js');
+
+            forked.send(JSON.stringify({
+                datasetId, standardId, importResult, handlerId, config: this.config,
+            }));
+
+            forked.on('message', async (response) => {
+                if (response.error) {
+                    await this.handleError(handlerId, response.error);
+                } else {
+                    this.logger.info(`Export complete for export handler_id: ${handlerId}`);
+                    await Models.handler_ids.update(
+                        {
+                            status: 'COMPLETED',
+                        },
+                        {
+                            where: {
+                                handler_id: handlerId,
+                            },
+                        },
+                    );
+                }
+                forked.kill();
+            });
+        }
+        return Command.empty();
```

Comment:
Wouldn't it make sense to schedule the cleaner command when the export is completed?

Reply:
You think that we should say that user can get export result only once?

Reply:
Off topic but could be useful. We can introduce caching by checking if the command result already exists in the handlers table.

Reply:
No, like it can call the result as many times as they want, but after an hour the result is deleted